src/python/pants/engine/native.py

```diff
@@ -314,14 +358,15 @@ def bootstrap_c_source(output_dir, module_name=NATIVE_ENGINE_MODULE):
     # (it kept the binary working) but inconvenient (it was relying on unspecified behavior, it meant
     # our binaries couldn't be stripped which inflated them by 2~3x, and it reduced the amount of LTO
     # we could use, which led to unmeasured performance hits).
-    def rename_symbol_in_file(f):
-      with open(f, 'r') as fh:
-        for line in fh:
-          if line.startswith('init{}'.format(module_name)) or line.startswith('PyInit_{}'.format(module_name)):
-            yield 'wrapped_' + line
-          else:
-            yield line
-    file_content = ''.join(rename_symbol_in_file(temp_c_file))
+    #
+    # We additionally remove the ifdefs that apply conditional `init` logic for Py2 vs Py3, in order
+    # to define a module that is loadable by either 2 or 3.
+    # TODO: Because PyPy uses the same `init` function name regardless of the python version, this
```

Comment:
To my knowledge we have never supported running on PyPy like pex does. Perhaps kill the TODO.